src/query.c

```diff
@@ -335,8 +335,7 @@ SDB_API char *sdb_querys (Sdb *r, char *buf, size_t len, const char *_cmd) {
 			SdbListIter *iter;
 			SdbKv *kv;
 			ls_foreach (list, iter, kv) {
-				//eprintf ("(%s)(%s)\n", kv->key, kv->value);
-				foreach_list_cb (&user, kv->key, kv->value);
+				foreach_list_cb (&user, SDBKV_KEY (kv), SDBKV_VALUE (kv));
```

Comment:
why this change i dont see a reason to have this macro anymore

Reply:
i understand the concern of having macros to make this easier to refactor, but this is making the code harder to read, what about just having 1 letter macros for that? K(kv) and V(kv)